firefox_ui_tests/functional/private_browsing/test_about_private_browsing.py

```diff
@@ -2,65 +2,54 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from marionette_driver import By
+from marionette_driver import By, Wait
 
 from firefox_ui_harness.decorators import skip_if_e10s
 from firefox_ui_harness import FirefoxTestCase
 
+from firefox_puppeteer.ui.windows import BrowserWindow
+
 
 class TestAboutPrivateBrowsing(FirefoxTestCase):
 
     def setUp(self):
         FirefoxTestCase.setUp(self)
-        self.pb_url = self.marionette.absolute_url('private_browsing/about.html?')
 
-    def tearDown(self):
-        self.marionette.close()
+        self.support_url = 'about:blank?'
+        self.pb_url = self.support_url + 'private-browsing'
+
+        self.prefs.set_pref('app.support.baseURL', self.support_url)
 
     @skip_if_e10s
```

Comment:
I know you didn't change this, but can we not do this in the manifest.ini file?  that is what we do for mochitests.